app/buildpacks/node_buildpack.rb

```diff
@@ -1,8 +1,8 @@
 class NodeBuildpack
   include Buildpack
 
-  LATEST    = '8'.freeze # 8 is the LTS version
-  SUPPORTED = %w(5 6 8 10).freeze
+  LATEST    = '12'.freeze # 8 is the LTS version
+  SUPPORTED = %w(5 6 8 10 11 12).freeze
```

Comment:
I would skip 11.  It isn't even active at the moment:

![schedule](https://raw.githubusercontent.com/nodejs/Release/master/schedule.svg?sanitize=true)

(Source: https://nodejs.org/en/about/releases/)